streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableKTableJoinTest.java

```diff
@@ -138,8 +138,8 @@ public void testJoin() throws Exception {
 
         // push all four items to the primary stream. this should produce four items.
 
-        for (int i = 0; i < expectedKeys.length; i++) {
-            driver.process(topic1, expectedKeys[i], "X" + expectedKeys[i]);
+        for (int expectedKey1 : expectedKeys) {
```

Comment:
Thank you for pointing out.
 just updated.
Let me know anything else needs to be fixed.